### ISSUE TITLE
It seems your project has been copied for personal profit.

### ISSUE THREAD

**Issue by shenlebantongying** [tuberry/ibus-tweaker#15]
See:
https://github.com/IBus-Customize/Customize-IBus/commit/03d3e083dbd3f78ac9d6525a2054d16451e43c96

https://github.com/openSUSE/mentoring/issues/158

If he is truly a free software user/contributor, he could just contribute to your repo or collaborate with you. He simply copy a few projects into one, so that he can put another "project" on his _résumé_. 

It doesn't violate GPL, but I feel a bit disgusted for this type of "rearrange a few buttons, then it is MY project" kind of shits (like many made-in-china "open source" or pirated software which are just renaming open source projects with little alternation. It is a corrupted culture.), and I don't think he will do anything about the project after GSoC ends. 

I think you may consider copying some functionality back since both are GPL-ed. It is still the same software but with some customization copied from some other projects (and renaming variables to increase "commit number" and SLOC like his other projects).

**Comment by tuberry** [comment]
Thanks for your kindness.

It's encouraging that someone is willing to contribute to open source, even if making a profit doesn't defeat that purpose. Sorry to say that I also *copied* some code from other projects in the early stage of development. :)

As for his project-specific features:
1. Adding background images under high-frequency changing text is distracting and confusing.It could be better to use border-image instead of background-image. In fact, if users want to write css files to customize them, I'd like to [recommend](https://github.com/tuberry/ibus-tweaker/issues/6) another extension — [user-theme-x](https://github.com/tuberry/user-theme-x). 
2. I have [implemented](https://github.com/tuberry/ibus-tweaker/issues/5#issuecomment-688061214) the input method indicator, but the effect is not good enough to reach the degree of fcitx5, that may need ibus upstream to improve the input context event mechanism. Maybe I will look into it later.
3. The pop-up animation seems to me to be a waste of tens or hundreds of milliseconds to see the results.
4. I'm not sure about the use case of drag & drop. I remember a kind of problem of linux input method is that the candidate box does not follow the cursor, is this to let the user manually make it follow? If so, that's not much of a user-friendly workaround.
5. For the rest, the added menu items are almost useless to me.

Of course the above are my own humble opinions, if there are any features you would like to see in a few of them or if you have any other suggestions, please open another issue so that we can discuss them.

Thanks again. ;)

**Comment by shenlebantongying** [comment]
Hahahahaha, I agree that the backgrounds suck hard too `:)`

**Comment by tuberry** [comment]
Another issue #16 has asked me to delete this issue but I think it is [necessary](https://github.com/tuberry/ibus-tweaker/issues/16#issuecomment-845892825) to ask your opinion. 

In order to avoid unnecessary arguments, if you do not reply within three working days, please allow me to assume that you agree to the deletion of this issue. Thanks. :)

By the way, the author @HollowMan6 of #16 would like to [communicate](https://github.com/tuberry/ibus-tweaker/issues/16#issuecomment-845912143) with you if possible. 
```
@HollowMan6 has a reply to you. But you blocked him, so he can't make any comments with this issue. You can unblock him and discuss with him by going to this issue:

https://github.com/HollowMan6/Customize-IBus/issues/5
```

**Comment by shenlebantongying** [comment]
Sorry for the trouble to you @tuberry. Was just going to tell you in hope of borrowing something back `:)`

Only the repo owner can delete issues completely, ​as it is supported by GitHub. Feel free to delete this one, don't want to waste time on useless arguments.

Cannot believe he writes that long, hahaha. I think I said it doesn't violate GPL right?

Don't feel like communicating with him as he keeps milking the community for his resume.
